Configuration/PyReleaseValidation/python/relval_gpu.py

```diff
@@ -71,4 +71,7 @@
 workflows[141.008512] = ['Run3-2023_JetMET2023B_RecoECALOnlyGPU',['RunJetMET2023B','HLTDR3_2023','RECODR3_reHLT_ECALOnlyGPU','HARVESTRUN3_ECALOnly']]
 workflows[141.008522] = ['Run3-2023_JetMET2023B_RecoHCALOnlyGPU',['RunJetMET2023B','HLTDR3_2023','RECODR3_reHLT_HCALOnlyGPU','HARVESTRUN3_HCALOnly']]
 #2023 HIon MC Patatrack pixel-only quadruplets on HydjetQ_MinBias_5362GeV_2023_ppReco on GPU (optional)
-workflows[160.502] = ['',['HydjetQ_MinBias_5362GeV_2023_ppReco','DIGIHI2023PPRECO','RAWPRIMESIMHI18','RECOHI2023PPRECOMB_PatatrackGPU','MINIHI2023PROD']]
\ No newline at end of file
+workflows[160.502] = ['',['HydjetQ_MinBias_5362GeV_2023_ppReco','DIGIHI2023PPRECO','RAWPRIMESIMHI18','RECOHI2023PPRECOMB_PatatrackGPU','MINIHI2023PROD']]
+
+# data 2023 Patatrack all GPU vs CPU validation
+workflows[141.583]=['Run3-2023_JetMET2023B_GPUValidation',['RunJetMET2023B','RecoData_Patatrack_AllGPU_Validation_2023','HARVESTData_Patatrack_AllGPU_Validation_2023']]
```

Comment:
this should likely be `141.008583`

Reply:
Few lines above there is also a:

https://github.com/cms-sw/cmssw/blob/0295cb18cb10e5774d20c2d910e4dc17ed352357/Configuration/PyReleaseValidation/python/relval_gpu.py#L74

should we rename this too? @cms-sw/pdmv-l2

Reply:
So the numbering seems to be done so that the first digits refers to the primary dataset (`Run3-2023_JetMET2023B`) or the sample (for HIon). 

I would keep this scheme and yes I would change also `160.502` accordingly (should be `160.03502`). But I can take care of that.

Reply:
I renamed the new workflow in https://github.com/cms-sw/cmssw/pull/42674/commits/f7eff8217ff3a143be76d185ba1f9280161cf013 and then also addressed `160.502` in https://github.com/cms-sw/cmssw/pull/42674/commits/3165d0dc7a263a0f2ae1c1d679c01c55c07b3b41

Reply:
Thanks